src/CounterState.php

```diff
@@ -4,25 +4,22 @@
 
 namespace Yiisoft\Yii\RateLimiter;
 
-/**
- * Rate limiter counter state.
```

Comment:
The description here was OK. I think we can expand it a bit:

Rate limiter counter state stores information about when the next request won't be limited.